Change from personal name to unit name in emails (#1439)
ScilifelabDataCentre/dds_web: dds_web/templates/mail/project_release.txt

> {% if unit_name is none %}
> You were added to this project by {{sender_name}}.
> {% else %}
> You were added to this project on behalf of {{unit_name}}{% if unit_email is not none %} ({{unit_email}}){% endif %}.
> {% endif %}

Reviewer: ```suggestion
{% if unit_email %}
You were added to this project on behalf of {{displayed_sender}} ({{unit_email}}).
{% else %}
You were added to this project by {{displayed_sender}}.
{% endif %}
```
